glide.yaml

```diff
@@ -149,6 +156,17 @@ import:
 - package: github.com/mjibson/esc
   version: ^0.1.0
 
+- package: github.com/gavv/monotime
```

Comment:
go1.9 added monotonic clock support so we don't need the `monotime` dep any more.